Template/PythonFastAPI/README.md

```diff
@@ -0,0 +1,116 @@
+# Power Skill API
+
+Sample PowerSkill API
+
+## Solution Flow
+
+
+## Environment variables
+
+The following sample environment variables need to be set for the process to work:
+
+```bash
+       
+                "TR_REGION": "westeurope", # Translator region
+                "DEBUG": "True", # Enable process debugging
+                "FLASK_DEBUG": "false", # Flask native debugging keep False
+                "FLASK_ENV": "development", # Toggle between development and production
+
+```
+
+## API Request/Response
+
+### API Inputs POST
+
+```json
+{
+    "values": [
+      {
+        "recordId": "100003490593495",
+        "data":
+           {
+             "fileContent": "[some content]",
+             "correlationId": "123233434334334",
+             "batch": "4535346534654654"
+           }
+      }
+    ]
+}
+```
+
+### API Outputs
+
+```json
+{
+    "values" :
+    [
+      {
+        "recordId": "100003490593495",
+        "correlationId": "123233434334334",
+        "batch": "4535346534654654",
+        "errors" : "",
+        "data" : {
+          "entities": [
+            {
+              "modelName" : "[modelname - e.g 1]",
+              "language": "AR",
+              "text" : [
+                 "ريك أستلي" ,"لورانس"
```

Comment:
Are you actually rickrolling us? :D

Reply:
Can we instead use the actual output from the sample as you'd get if you ran it as is?

Reply:
Implemented